Configuration/PyReleaseValidation/python/relval_2026.py

```diff
@@ -26,7 +26,9 @@
 numWFIB.extend([22434.0]) #2026D92
 numWFIB.extend([22834.0]) #2026D93
 numWFIB.extend([23234.0]) #2026D94
-numWFIB.extend([23634.0]) #2026D95
+numWFIB.extend([23634.0,23634.103]) #2026D95 (NoPU, with aging)
```

Comment:
Hi @srimanob , could you please add also a `23634.21` with prod-like PU conditions? This is needed for profiling reco jobs

Reply:
Done.